src/ploneintranet/workspace/browser/tiles/workspaces.py

```diff
@@ -49,7 +49,7 @@ def my_workspaces(context):
             })
         except Unauthorized:
```

Comment:
This whole try/except can now be dropped - that was one of the goals of this story, to be able to rely on the portal_catalog security filtering.